models/staging/edfi_3/stage/stg_ef3__discipline_actions__disciplines.sql

```diff
@@ -10,6 +10,7 @@ flattened as (
         k_student,
         k_student_xyear,
         {{ extract_descriptor('value:disciplineDescriptor::string') }} as discipline_type
+        {{ extract_extension(model_name=this.name, flatten=True) }}
```

Comment:
i think we need to first pull out v_ext from value, a la https://github.com/edanalytics/edu_edfi_source/blob/55b874cb5b108cb18cb823fc62469d36c5a586da/models/staging/edfi_3/stage/stg_ef3__stu_spec_ed__program_services.sql#L23